src/Symfony/Component/Validator/Constraints/File.php

```diff
@@ -49,13 +51,15 @@ class File extends Constraint
 
     public $binaryFormat;
     public $mimeTypes = [];
+    public int|null $filenameMaxLength = null;
     public array|string|null $extensions = [];
     public $notFoundMessage = 'The file could not be found.';
     public $notReadableMessage = 'The file is not readable.';
     public $maxSizeMessage = 'The file is too large ({{ size }} {{ suffix }}). Allowed maximum size is {{ limit }} {{ suffix }}.';
     public $mimeTypesMessage = 'The mime type of the file is invalid ({{ type }}). Allowed mime types are {{ types }}.';
     public string $extensionsMessage = 'The extension of the file is invalid ({{ extension }}). Allowed extensions are {{ extensions }}.';
     public $disallowEmptyMessage = 'An empty file is not allowed.';
+    public string $filenameTooLongMessage = 'The filename is too long. Allowed max filename is {{ filename_max_length }} characters.';
```

Comment:
I prefer the message in the description:
The filename is too long. It should have 50 character or less.

Reply:
Changed ! Thank you :+1: